packages/BetterPhpDocParser/PhpDoc/ArrayItemNode.php

```diff
@@ -13,31 +13,21 @@ final class ArrayItemNode implements PhpDocTagValueNode, Stringable
 {
     use NodeAttributes;
 
-    /**
-     * @param String_::KIND_*|null $kindValueQuoted
-     */
     public function __construct(
         public mixed $value,
-        public mixed $key,
-        public int|null $kindValueQuoted = null,
-        public int|null $kindKeyQuoted = null,
+        public mixed $key = null,
```

Comment:
Looks like a BC break

Reply:
@staabm good catch! added them back as deprecated

Reply:
@TomasVotruba just to confirm I guess I can remove them in the end, not need for deprecation?